src/firebolt_ingest/table_model.py

```diff
@@ -31,12 +31,15 @@ def parse_yaml(cls, yaml_obj: Any):
     "VARCHAR",
     "STRING",
     "DATE",
+    "PGDATE",
     "DATETIME",
     "TIMESTAMP",
+    "TIMESTAMPNTZ",
+    "TIMESTAMPTZ",
     "BOOLEAN",
 }
 
-DATE_TIME_TYPES = {"DATE", "TIMESTAMP", "DATETIME"}
+DATE_TIME_TYPES = {"DATE", "DATETIME", "TIMESTAMP", "TIMESTAMPNTZ", "TIMESTAMPTZ"}
```

Comment:
TIMESTAMPTZ can't be in partition statement, only after casting it to UTC https://firebolt-analytics.slack.com/archives/C02SMTT26VA/p1692348071841589?thread_ts=1692336345.878089&cid=C02SMTT26VA and https://firebolt-analytics.slack.com/archives/C02SMTT26VA/p1692281190659239